test/conversions.jl

```diff
@@ -3,6 +3,11 @@ import Compat.Dates
 utc = FixedTimeZone("UTC")
 warsaw = resolve("Europe/Warsaw", tzdata["europe"]...)
 
+# Converting from the stdlib UTC
+@test FixedTimeZone(UTC()) == utc
```

Comment:
I'm not sure I want to support using an instance of `UTC` as this is never actually utilized in Base. In fact they only ever seem to use the type so maybe the type should be redefined as abstract or rethought.

Reply:
Your interface currently says it accepts any `TimeZone` :P

Reply:
Fair. This happens to be the only `TimeZone` I don't like 😆